cfn/paws-collector.template

```diff
@@ -144,6 +144,11 @@
             "Type": "String",
             "Default": "true",
             "AllowedValues":["true"]
+        },
+        "ExistingDeadLetterQueueArn":{
```

Comment:
this template is intended to create all the resources. In this case create DLQ sqs and reference it's ARN in `PawsPollStateQueue`
I think for the sake of time you can just add a TODO item to add DLQ into this template because it doesn't look like anyone is going to use it.